### PR TITLE
feat: Update Avatar and Profile Page Loading State

### PR DESCRIPTION
**Pull request by Golodhros**
### Summary of Changes
Updates Avatar to have a shimmering circle while loading
Updates the header of the Profile page to have a shimmering loader while loading

-_Amundsen_Profile

### Tests
Updated tests

### CheckList
Make sure you have checked **all** steps below to ensure a timely review.
- [x] PR title addresses the issue accurately and concisely. Example: "Updates the version of Flask to v1.0.2"
    - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
- [x] PR includes a summary of changes, including screenshots of any UI changes. 
- [x] PR adds unit tests, updates existing unit tests, __OR__ documents why no test additions or modifications are needed.
- [x] In case of new functionality, my PR adds documentation that describes how to use it.
    - All the public python functions and the classes in the PR contain docstrings that explain what it does
- [x] PR passes all tests documented in the [developer guide](https://github.com/lyft/amundsenfrontendlibrary/blob/master/docs/developer_guide.md#testing)

### CODE REVIEW

**Comment by ttannis** [comment]
@Golodhros are you able to clean up the diff to remove changes that are already merged down to master?

I can also look at each relevant commit separately and see how that goes.

**Comment by Golodhros** [comment]
> @Golodhros are you able to clean up the diff to remove changes that are already merged down to master?
> 
> I can also look at each relevant commit separately and see how that goes.

Done! I forgot to set the upstream to master.

File: amundsen_application/static/css/_avatars.scss [hunk 1]
```diff
@@ -1,7 +1,3 @@
-.sb-avatar > div {
```
Comment (ttannis) [inline amundsen_application/static/css/_avatars.scss:1]: I would only want to apply this in the `NavBar` from the perspective that the `NavBar` usage is the only one I'm aware was discussed, and the white border is useful in other places like in the FrequentUsers UI where we stack these on top of one another.

Comment (Golodhros) [reply amundsen_application/static/css/_avatars.scss:1]: I see, I will retrieve it for the FrequentUsers UI.

**Pull request merged**